modules/installation-special-config-kargs.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-special-config.adoc
+
+[id="installation-special-config-kargs_{context}"]
+
+= Adding day-1 kernel arguments
+You can add kernel arguments to all master and/or worker nodes during initial cluster
+installation or as a day-2 activity (preferred). Here are some reasons you might want
+to add kernel arguments during cluster installation so they take effect before
+the systems first boot up:
+
+* You want to disable a feature, such as SELinux, so it has no impact on the systems when they first come up
+* You need to do some low-level network configuration before the systems start
+
+To add kernel arguments to master or worker nodes, you can create a MachineConfig object
+and inject that object into the set of manifest files used by Ignition during
+cluster setup, as described below.
+
+For a listing of arguments you can pass to a RHEL 8 kernel at boot time, see
+link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/performing_an_advanced_rhel_installation/index#kickstart-and-advanced-boot-options_installing-rhel-as-an-experienced-user[Boot options].
+Again, you should only add kernel arguments with this procedure if they are needed to complete the initial
+{product-title} installation.
+
+.Procedure
+
+. Generate the Kubernetes manifests for the cluster:
++
+----
+$ ./openshift-install create manifests --dir=<installation_directory> 
+----
+
+. Decide if you want to add kernel arguments to worker or master nodes.
+
+. In the `openshift` directory, create a file (for example,
+99_openshift-machineconfig_master-kargs.yaml) to define a MachineConfig
+object to add the kernel settings. Here's what the contents of that file
+could look like if you were adding a `loglevel=7` kernel argument to
```

Comment:
s/Here's what the contents of/This is what the contents of

Reply:
Sure. I did this: "This content adds a `loglevel=7` kernel argument to master nodes:"